### ISSUE TITLE
Forma de Instalación

### ISSUE THREAD

**Issue by X-PLORADOR** [tonitim13/icloudpos#1]
Hola, r}sera que me puedes explicar como instalrlo, ya que no encuentro la base de datos .sql

**Comment by tonitim13** [comment]
Hola, como estas. como para que quieres usar el codigo. podria ayudarte. Si quieres escribame a info@teayudo.pe
Saludos Mortal

**Comment by tonitim13** [comment]
Pero sabes para qué es ese sistema?

**Comment by X-PLORADOR** [comment]
pues por lo que vi, me pareció ver que era un Punto de venta (pos), que es lo mismo a un TPV (terminal punto de venta)

**Comment by X-PLORADOR** [comment]
es decir, tu proyecto es un Punto de Venta (POS) que es igual a una terminal punto de venta (TPV)

**Comment by tonitim13** [comment]
Efectivamente. Ese sistema pertenece a un grupo de desarrollo, lo que vez allí es una versión demo. Sí está interesado en adquirir el sistema o tiene alguna duda puede contactar a nuestro comercial.
info@teayudo.pe
WhatsApp: +51950330576

**Comment by X-PLORADOR** [comment]
ah, es demo, como no dice en la carpeta.. ok

**Comment by tonitim13** [comment]
Ok. Ya le dejé los contactos en caso estar interesado
